database-schema/create_billing_and_fraud_events_table.sql

```diff
@@ -0,0 +1,28 @@
+CREATE SCHEMA billing AUTHORIZATION postgres;
+
+CREATE TABLE billing.billing_events
+(
+    time_stamp                   timestamp,
+    session_id                   text COLLATE pg_catalog."default",
+    hashed_persistent_id         text COLLATE pg_catalog."default",
+    request_id                   text COLLATE pg_catalog."default",
+    idp_entity_id                text COLLATE pg_catalog."default",
+    minimum_level_of_assurance   text COLLATE pg_catalog."default",
+    required_level_of_assurance  text COLLATE pg_catalog."default",
+    provided_level_of_assurance  text COLLATE pg_catalog."default"
+)
+TABLESPACE pg_default;
+ALTER TABLE billing.billing_events OWNER to postgres;
+
+CREATE TABLE billing.fraud_events
+(
+    time_stamp            timestamp,
+    session_id            text COLLATE pg_catalog."default",
+    hashed_persistent_id  text COLLATE pg_catalog."default",
+    request_id            text COLLATE pg_catalog."default",
+    entity_id             text COLLATE pg_catalog."default",
+    fraud_event_id        text COLLATE pg_catalog."default",
+    fraud_indicator       text COLLATE pg_catalog."default"
+)
+TABLESPACE pg_default;
+ALTER TABLE billing.fraud_events OWNER to postgres;
```

Comment:
Could you please rename these to `V1__create_audit_events_table.sql` and `V2__create_billing_and_fraud_events_table.sql` as Flyway expects the prefix 🙂

Reply:
Done.